packages/admin/src/services/SchemeService.test.ts

```diff
@@ -177,4 +179,44 @@ describe('SchemeService', () => {
       );
     });
   });
+
+  describe('schemeApplicationIsInternal', () => {
+    it('should return false when the scheme does not have an internal application form', async () => {
+      const schemeId = 'schemeId';
+      const sessionId = 'sessionId';
```

Comment:
These two variables could be set outside of this test because they're the same in both tests